### PR TITLE
Feature: Updated the fallback color for Acrylic

### PR DESCRIPTION
**Pull request by heftymouse**
This PR changes the Acrylic fallback colour on dark mode to `#202020` which is a much more pleasing colour also used by Mica.

**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by yaira2** [comment]
> It also changes the sidebar and top bar border to use the divider stroke brush making it more consistent.

Can you separate this change? It needs to be discussed with design and I don't want to hold back the other improvements.

File: src/Files.App/Helpers/AppSystemBackdrop.cs [hunk 1]
```diff
@@ -88,11 +103,36 @@ private void OnSettingChanged(object? sender, Shared.EventArguments.SettingChang
...
+					var acrylicController = new DesktopAcrylicController();
+					SetAcrylicBackdropProperties(acrylicController, theme);
+					return acrylicController;
 
 				default:
 					return null;
 			}
 		}
+
+		private void SetAcrylicBackdropProperties(DesktopAcrylicController controller, SystemBackdropTheme theme)
```
Comment (yaira2) [inline src/Files.App/Helpers/AppSystemBackdrop.cs:115]: What happens if the color mode is changed at run time?

Comment (heftymouse) [reply src/Files.App/Helpers/AppSystemBackdrop.cs:115]: It changes automatically, see https://github.com/files-community/Files/blob/c5a627f6012e8c5e664d6e84ca567ad0e643ce9e/src/Files.App/Helpers/AppSystemBackdrop.cs#L54

**Review by yaira2 — DISMISSED** [review]

File: src/Files.App/Helpers/AppSystemBackdrop.cs [hunk 2]
```diff
@@ -88,11 +103,36 @@ private void OnSettingChanged(object? sender, Shared.EventArguments.SettingChang
...
+		{
+			// need to set all properties to work around a bug where other properties stop updating when fallback color is changed
+			// this uses the Thin Acrylic recipe from the WinUI Figma toolkit
```
Comment (0x5bfa) [inline src/Files.App/Helpers/AppSystemBackdrop.cs:118]: Please capitalize. Otherwise, LGTM.

**Review by 0x5bfa — APPROVED** [review]

**Pull request merged**